topdown/glob.go

```diff
@@ -15,7 +15,7 @@ const globCacheMaxSize = 100
 var globCacheLock = sync.Mutex{}
 var globCache map[string]glob.Glob
```

Comment:
Can we drop this? I'd expect that if `bctx.GlobBuiltinCache == nil`, then we'd simply not use a cache.

Reply:
It would be good to keep the existing behavior as a fallback imo.

Reply:
For `http.send` we fall back to the intra-query cache if the inter-query is not set.

Reply:
The old cache is still inter-query, though. It's harmless to keep, however, so I won't push back hard on this 😄. 
Having no cache for compiled globs and regexes is harmless too, though, it just comes with a small performance hit.